src/App.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+export const goodsFromServer: string[] = [
```

Comment:
```suggestion
export const goodsFromServer = [
```

It's not really needed